testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/group/GroupTest.java

```diff
@@ -839,6 +841,67 @@ public void adminEndpointAccessibleWhenAdminRoleAssignedToGroup() {
         }
     }
 
+    /**
+     * Groups search with query returns unwanted groups
+     * @link https://issues.redhat.com/browse/KEYCLOAK-18380
+     */
+    @Test
+    public void searchForGroupsShouldOnlyReturnMatchingElementsOrIntermediatePaths() {
+        //Map Group Provider is not working the same way as JPA Group Provider
+        //see https://issues.redhat.com/browse/KEYCLOAK-18390
+        Assume.assumeTrue(Profile.getDisabledFeatures().contains(Profile.Feature.MAP_STORAGE));
```

Comment:
Please remove. The same test has to work for map storage

Reply:
@hmlnarik I just removed the `Assume` line and with the recent fixes in the Map-store it seems to work fine now.